test/parallel/test-http-status-code.js

```diff
@@ -23,13 +23,15 @@
 require('../common');
 const assert = require('assert');
 const http = require('http');
+const Countdown = require('../common/countdown');
 
 // Simple test of Node's HTTP ServerResponse.statusCode
 // ServerResponse.prototype.statusCode
 
 let testsComplete = 0;
 const tests = [200, 202, 300, 404, 451, 500];
 let testIdx = 0;
+const countdown = new Countdown(tests.length, () => s.close());
 
 const s = http.createServer(function(req, res) {
   const t = tests[testIdx];
```

Comment:
This can be removed and then the `assert.strictEqual()` can compare to the top scope `test` variable.